MetadataExtractor/Formats/Exif/makernotes/OlympusFocusInfoMakernoteDescriptor.cs

```diff
@@ -107,15 +107,13 @@ public sealed class OlympusFocusInfoMakernoteDescriptor(OlympusFocusInfoMakernot
             if (values.Length == 0)
                 return null;
 
-            var join = $"{values[0]}" + (values.Length > 1 ? $"{values[1]}" : "");
-
-            return join switch
+            return (values[0], values.Length > 1 ? values[1] : -1) switch
             {
-                "0" => "Off",
-                "1" => "On",
-                "0 0" => "Off",
-                "1 0" => "On",
-                _ => "Unknown (" + join + ")",
+                (0, -1) => "Off",
+                (1, -1) => "On",
+                (0, 0) => "Off",
+                (1, 0) => "On",
+                _ => $"Unknown ({string.Join(" ", values)})"
```

Comment:
-1 is an in-band value for the tag. How about this?

```c#
return (values[0], values.Length > 1 ? (int?)values[1] : null) switch
{
    (0, 0 or null) => "Off",
    (1, 0 or null) => "On",
    _ => $"Unknown ({string.Join(" ", values)})"
}
```

Reply:
Oh just seen your comment about -1 being unsigned. Am wondering if it becomes `ushort.MaxValue`. Checking.

Reply:
Yeah, seems ok, ignore :)

https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBBmABM3AYVwG9NcL88EUAGXAWQAoBXAZwAsB7AJwBcA2gF1cEAJSlylaQgDsuJhAG0hAGlEA6ADIBTGAHM+HXAB5cSXAH5cAWhS4QogSiES2Adyh8AxhynSKMgwAkIVadVoJAF4APlwAIgBBeNV/UIoAfVxYhIAheLTpAF8AbjSizCKgA===